terraform/ec2/win/main.tf

```diff
@@ -213,6 +237,42 @@ resource "null_resource" "integration_test_run_validator" {
   }
 }
 
+resource "null_resource" "integration_test_run_validator_userdata" {
+  # run validator only when test_dir is not passed e.g. the default from variable.tf
+  count = length(regexall("/feature/windows/userdata", var.test_dir)) > 0 ? 1 : 0
+  depends_on = [
+    null_resource.integration_test_setup_validator,
+    null_resource.integration_test_wait,
+  ]
+
+  connection {
```

Comment:
If we are using EC2 "userdata" to launch the instance and install the agent, then do we still need winrm access to the instance?

In other words, after launching a Windows EC2 instance with the agent installed with user-data, what additional steps occur?

Reply:
This runs the validator, it does not reinstall / restart the agent. The validator runs on the ec2 instance.